src/elements/types/RestElement.js

```diff
@@ -0,0 +1,27 @@
+import Expression from '../Expression';
+
+export default class RestElement extends Expression {
+
+    // TODO: Requires a function?
+
+    constructor(childNodes) {
+        super('RestElement', childNodes);
+    }
+
+    _acceptChildren(children) {
+        let argument = null;
+
+        children.passToken('Punctuator', '...');
+        argument = children.passNode('Identifier');
```

Comment:
Is it possible to have whitespace between `...` and `Identifier`? If yes, then you should add `children.skipNonCode();` between.